src/views/Services.md

```diff
@@ -0,0 +1,51 @@
+<div class="container-fluid monarch-view services">
+    <h2 class="page-title">Monarch Web Services</h2> 
+    <div class="row architecture">
+        <div class="col-6">
+            <img class="services" src="../assets/img/Monarch-Architecture2.png"/>
+        </div>
+        <div class="col-6 text">
+        <h5><strong>Architecture</strong></h5>
+        <p>Structured and unstructured data sources are loaded into SciGraph via dipper (1) , 
+        our data ingest pipeline. A variety of ontologies are processed in the Monarch Integration Layer 
+        and are also loaded into SciGraph, resulting in a combined knowledge and data graph (2). Data is 
+        disseminated via SciGraph Services, wrapped around by the BioLink API (3). This lightweight API 
+        makes few assumptions, acting as a data access layer for all of Monarch’s Services. Users can 
+        interrogate Monarch using our tools for text annotation, navigating the ontologies, and matching 
+        phenotype profiles (4).</p>
+        </div>
+    </div>
+    <div class="row biolink">
```

Comment:
This looks great. Thanks!

A couple of things to fix: 
- Please add the word '_substances_' back in front of the line listing '_small molecules, drugs, chemical entities_'
- Since you are going back in, please add Oxford commas where appropriate. :bowtie: - I only just noticed the typos I ported over from the BL doc pages: 
  - ```traits, and clinical measurements```
  - ``` clinical trials, and 'natural experiments'```

Reply:
Please note that it should be 'substances:' <-- include a colon punctuation mark as well, please.